.editorconfig

```diff
@@ -0,0 +1,5 @@
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
@delgurth Could you please either update the `indent_size` in `.editorconfig` to match the existing code or remove the file altogether, and revert the whitespace related changes in this PR?

Reply:
I'll remove all formatting changes. I've tried to align it, but if I recall correctly the formatting was different in different parts of the codebase.